guides/common/modules/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -76,14 +77,76 @@ ifdef::katello[]
 # foreman-rake katello:upgrade_check
 ----
 endif::[]
+ifdef::foreman-deb[]
+. Update repositories in `/etc/apt/sources.list.d/foreman.list` on your {ProjectServer} to consume content for {ProjectVersion}:
++
+[options="nowrap" subs="attributes"]
+----
+deb http://deb.theforeman.org/ _My_Distribution_Codename_ {ProjectVersion}
+deb http://deb.theforeman.org/ plugins {ProjectVersion}
+----
++
+Replace _My_Distribution_Codename_ with "bullseye" if you run {ProjectServer} on Debian 11, "jammy" for Ubuntu 22.04, and "focal" for Ubuntu 20.04.
+// List of supported OS for foreman-deb has to match "guides/common/modules/proc_configuring-repositories-foreman-deb.adoc"
+. Update to {Project} {ProjectVersion}:
++
+[options="nowrap" subs="attributes"]
+----
+# apt-get update
+# apt-get --only-upgrade install ruby\* foreman\*
+----
+. Migrate the database on {ProjectServer}:
```

Comment:
Should I drop "on {ProjectServer}" everywhere because it's obvious?

Reply:
All the steps should be performed on {ProjectServer}, is that correct? You could just write it once at the very beginning:

```
.Upgrade {ProjectServer}
On your {ProjectServer}:

<step 1>
```

Reply:
Dropped "on {ProjectServer}"; the procedure is named ".Upgrade {ProjectServer}".